spring-cloud-commons/src/main/java/org/springframework/cloud/client/loadbalancer/reactive/DefaultRequestContext.java

```diff
@@ -29,9 +31,18 @@
 	 */
 	private String hint = "default";
 
+	/**
+	 * The original load-balanced request.
+	 */
+	private Object originalRequest;
```

Comment:
In our usage via the `BlockingLoadBalancerClient` and `ReactorLoadBalancerExchangeFilterFunction` it's actually always `ClientResponse`, but on the interface level, it could be any object. Have tried using a generic value here, but due to similar issues as [in `CompletionContext`](https://github.com/spring-cloud/spring-cloud-commons/pull/733/files#r434712895), have settled for `Object`.